azure-pipelines.yml

```diff
@@ -26,23 +26,19 @@ jobs:
   pool:
     name: NetCore1ESPool-Public
     demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre.Open
-  timeoutInMinutes: 40
+  timeoutInMinutes: 120
```

Comment:
needed to increase the timeout because in the event that we do have a crash it can take a long time to upload these dump files.  Tested with `--blame-crash-collect-always` so we got dump of every single test assembly and 120 minutes was enough for all 75 dump files to be uploaded.